ios/App/App/FsWatcher.swift

```diff
@@ -45,7 +45,7 @@ public class FsWatcher: CAPPlugin, PollingWatcherDelegate {
         call.resolve()
     }
 
-    public func recevedNotification(_ url: URL, _ event: PollingWatcherEvent, _ metadata: SimpleFileMetadata?) {
```

Comment:
@andelf Please check this change.

Reply:
That's OK. this is not exported anywhere else.